contrib/conan/recipes/breakpad/conanfile.py

```diff
@@ -0,0 +1,66 @@
+from conans import ConanFile, CMake, MSBuild, tools
+import os
+import shutil
+
+
+class BreakpadConan(ConanFile):
+    description = "Breakpad tools for symbol uploading"
+    name = 'breakpad'
+    version = "216cea7b"
+    license = 'https://chromium.googlesource.com/breakpad/breakpad/+/master/LICENSE'
+    url = 'https://gitlab.com/ArsenStudio/ArsenEngine/dependencies/conan-breakpad'
+    generators = 'cmake'
+    branch = 'master'
+    exports = ["CMakeLists.txt", "fix-unique_ptr.patch"]
+    build_requires = "libdisasm/0.23@orbitdeps/stable"
+    
+    settings = 'os', 'os_build', 'arch_build', 'compiler', 'arch'
+    options = {
+        "fPIC": [True, False]
+    }
+    default_options = {
+        "fPIC": True
+    }
+    
+    _source_dir = "breakpad"
+
+    def source(self):
+        self.run( 'git clone https://chromium.googlesource.com/breakpad/breakpad')
+        if self.settings.os == 'Linux':
+            self.run(
+                'git clone https://chromium.googlesource.com/linux-syscall-support breakpad/src/third_party/lss')
+        self.run('git checkout {}'.format(self.version), cwd='breakpad/')
+        shutil.move('CMakeLists.txt', self._source_dir)
+        tools.patch(patch_file="fix-unique_ptr.patch",
+                    base_path=self._source_dir)
+
+    def build(self):
+        cmake = CMake(self)
+        
+        if self.settings.os_build == "Windows":
+            vcvars = tools.vcvars_dict(self.settings)
+            cmake.definitions["VSINSTALLDIR"] = vcvars["VSINSTALLDIR"].rstrip('\\')
+
+        cmake.configure(source_folder=self._source_dir)
+        cmake.build()
+        
+    def copy_sym_upload_tools(self):
+        if self.settings.os == "Windows":
```

Comment:
`self.settings.os_build` here or use wildcards for the copy command

Reply:
Fixed